### PR TITLE
feat: add meta description

### LINKED ISSUE #141: add better meta description

### PR DESCRIPTION
**Pull request by creisle**
closes: #141

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
+
+<head>
```
Comment (areisle) [inline public/index.html]: why are all the indents changed?

Comment (creisle) [reply public/index.html]: b/c I have autoformat on save = true and this is how vscode formats html

Comment (creisle) [reply public/index.html]: what settings do you want to use for html formatting?

Comment (creisle) [reply public/index.html]: added inner indent setting

```
    "html.format.indentInnerHtml": true
```

**Review by areisle — APPROVED** [review]

**Pull request merged**